.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: go
+go: "1.13"
+
+services:
+  - docker
+
+install: true
+script:
+  - make docker-test
```

Comment:
We already do this on the prow run. Please drop it.

Reply:
Done